### DISCUSSION TITLE
Is this library compatible with Java 8?

### DISCUSSION THREAD

**Discussion by fungastorba** [Yubico/java-webauthn-server#372] [Q&A] [answered]
I couldn't find anywhere a minimum requirement for the Java version.

**Comment by emlun** [comment, marked as answer]
Yes, Java 8 is [one of the CI environments](https://github.com/Yubico/java-webauthn-server/actions/runs/10343823181/job/28628479040) and we also [test that the releases (compiled with JDK 17) are backwards-compatible with Java 8](https://github.com/Yubico/java-webauthn-server/blob/240b8d937e5aeb6d861625dada77035b49b839bf/.github/workflows/build.yml#L47-L54).
